scripts/tests/testUnit.sh

```diff
@@ -4,4 +4,4 @@
 export NODE_ENV='test'
 
 echo 'Start firestore emulator and run unit tests:'
-firebase emulators:exec 'nyc mocha test/unit/**'
+firebase emulators:exec 'nyc --n=models --n=middlewares/**.js --n=services --n=utils mocha test/unit/**'
```

Comment:
Tried using array here to include multiple folders but couldn't get syntax right.